docs/configuration.md

```diff
@@ -1664,6 +1664,15 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
+        <td>0</td>
+        <td>
+            A port number hen connecting with SSL. Default value 0 means to be determined automatically.
```

Comment:
There are a few typos here, and I don't think you need to repeat the point about multiple services. I'd suggest changing the prop name above to `spark.ssl.port` and then:

Port number to listen on for SSL connections. Default value of 0 means the port will be determined automatically.